src/notation/view/notationviewinputcontroller.cpp

```diff
@@ -333,6 +419,13 @@ void NotationViewInputController::mousePressEvent(QMouseEvent* event)
         return;
     }
 
+    if (viewInteraction()->isHitGrip(logicPos)) {
+        viewInteraction()->startEditGrip(logicPos);
+        return;
+    }
+
+    m_pressedOnSelected = hitElement && hitElement == viewInteraction()->selection()->element();
```

Comment:
needed to allow clicking on an already selected text element to enter text edit mode (as per MU3)